coalib/bears/Bear.py

```diff
@@ -12,6 +12,7 @@
 from coala_utils.decorators import (enforce_signature, classproperty,
                                     get_public_members)
 
+# Ignore PyUnusedCodeBear
 from coalib.bears.requirements.PackageRequirement import PackageRequirement
 from coalib.bears.requirements.PipRequirement import PipRequirement
```

Comment:
`from coalib.bears.requirements.PipRequirement import PipRequirement` This line also needs to be commented as ignore too(if I recall)